build/bin/build-server-rpm

```diff
@@ -46,6 +46,12 @@ prepare_specs() {
         local full_commit=$(git -C "${git_dir}" rev-parse HEAD)
         local short_commit=${full_commit:0:7}
 
+        if [[ ${spec_name} -eq "pmm-server" ]]; then
+            local pmm_branch=$(grep -A 3 "\[submodule \"pmm\"\]" .gitmodules | grep "branch" | awk '{print $3}')
+            local pmm_commit=$(git ls-remote git://github.com/Percona/pmm.git | grep refs/heads/${pmm_branch} | awk '{print $1}')
```

Comment:
We should not use remote commit – it can be different from the one in submodule.
I think `git rev-parse HEAD` could work there.

Reply:
The pmm submodule isn't used in the regular way. No new pmm.rpm is created.
It's only used for a parameter branch and based on this parameter the swagger folder is fetched from the repo.
It has been added the flag --ref just in case.

Reply:
Thanks for explaining how it works. That is still wrong.
percona/pmm is added as a submodule to this repository. git submodule always references a specific commit. If we have to use that commit for something, we should use that commit, not commit at the remote repository that could be different at the time we use it.

Reply:
It has been replaced by "git rev-parse HEAD"